### PR TITLE
deps: replace webextension-polyfill-ts with webextension-polyfill + @types/webextension-polyfill

### PR DESCRIPTION
**Pull request by legobeat**
This is a follow-up on #37 .

As can be seen in the lockfile diff, this implies a bump of `webextension-polyfill` from `0.8.0` to `0.10.0`.

Keeping webextension-polyfill as a runtime dependency is intended as an intermediary measure to keep this change non-breaking, making it a peerDependency in a future update.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,10 +31,11 @@
   },
   "homepage": "https://github.com/MetaMask/extension-port-stream#readme",
   "dependencies": {
-    "webextension-polyfill-ts": "^0.26.0"
+    "webextension-polyfill": ">=0.10.0"
   },
   "devDependencies": {
     "@metamask/eslint-config": "^4.1.0",
+    "@types/webextension-polyfill": ">=0.10.0",
```
Comment (Gudahtt) [inline package.json:38]: Nit: Could we use a `^` range here instead? Seems more conventional and avoids post-v1 breaking changes in theory.


```suggestion
    "webextension-polyfill": "^0.10.0"
  },
  "devDependencies": {
    "@metamask/eslint-config": "^4.1.0",
    "@types/webextension-polyfill": "^0.10.0",
```

**Review by Gudahtt — DISMISSED** [review]
LGTM, though I did have one suggestion

File: package.json (hunk 1 above)
Comment (Gudahtt) [reply package.json]: This seems OK to me, though I'm not sure why this would be preferable to the more conventional `^0.10.0` range

**Review by Gudahtt — APPROVED** [review]

**Review by mcmire — APPROVED** [review]
Makes sense.

**Pull request merged**